src/commands/misc/barcode.js

```diff
@@ -0,0 +1,53 @@
+const { Command, CommandError, SwitchbladeEmbed, Constants } = require('../../')
+
+module.exports = class UPCBarcode extends Command {
+  constructor (client) {
+    super({
+      name: 'barcode',
+      aliases: ['upc', 'upcbarcode', 'upcbarc'],
+      category: 'misc',
+      requirements: {
+        apis: [ 'barcode' ]
+      },
+      parameters: [{
+        type: 'string',
+        full: true,
+        missingError: 'commands:barcode.noCode'
+      }]
+    }, client)
+  }
+
+  async run ({ t, channel }, code) {
+    channel.startTyping()
+    try {
+      const { data } = await this.client.apis.barcode.request(code)
+      const embed = new SwitchbladeEmbed()
+        .setTitle(data.title)
+        .setColor(Constants.BARCODE_COLOR)
+        .setFooter(
+          t('commands:barcode.footer'),
+          'https://upcdatabase.org/images/logo.png'
+        )
+      if (data.description !== '') embed.setDescription(`/${data.description}/`)
+      if (data.images) embed.setThumbnail(data.images[0])
+      if (data.stores) {
+        embed.addFields(
+          data.stores.map(d =>
+            ({
+              name: d.store,
+              price: `${d.price ? `$${d.price}` : ''}`
+            })
+          )
+        )
+      }
+      channel.send(embed).then(() => channel.stopTyping())
+    } catch (err) {
+      channel.stopTyping()
+      if (err.response && err.response.status === 404) {
+        throw new CommandError(t('commands:barcode.notFound'))
```

Comment:
It isn't throwing this when no information is found
![image](https://user-images.githubusercontent.com/27602189/100764981-bef96c80-33d5-11eb-838b-b192c1376ae6.png)
[data result](https://github.com/SwitchbladeBot/switchblade/pull/1255/files#diff-89d9a38baa2a1b2e2a0aa1f8aa310d8848e1ba5ff746baaa9b0d603a5ccf993aR23):
![image](https://user-images.githubusercontent.com/27602189/100765011-c7ea3e00-33d5-11eb-8674-02124d49c416.png)